README.md

```diff
@@ -46,7 +46,7 @@ Flutter code is powered by the world-class [Dart platform], which enables
 compilation to 32-bit and 64-bit ARM machine code for iOS and Android, as well
 as JavaScript for the web and Intel x64 for desktop devices.
 
-![Dart platforms][Dart platforms]
+![Dart platform diagram][]
```

Comment:
```suggestion
![Dart platform diagram][Dart platform diagram]
```

Reply:
My understanding is that this is an implicit link name shortcut, and that the two are functionally identical. Based on inspection of the output, both of these two produce identical alt tags of "Dart platform diagram".

Reply:
Ah, I couldn't tell from running through the current read me with voiceover. The diff page read out a link, but I'm guessing that's different since it's generated by GitHub.
Good to know, thanks! 🙂